configs/dist-git/dist-git.conf

```diff
@@ -7,6 +7,7 @@ git_author_email = rel-eng@lists.fedoraproject.org
 
 lookaside_dir  = /var/lib/dist-git/cache/lookaside/pkgs
 gitroot_dir    = /var/lib/dist-git/git
+git_gc_depth   = 3
```

Comment:
Please just quickly document what it is (it relates to the timer, script, etc., what the number means, relation to `gitroot_dir`).

Reply:
We, in Copr, have depth 3. But I believe that the standard dept is two. If that is true, I will prefer to have 2 here.

Reply:
I changed the number to 2.